Week1/homework/app.js

```diff
@@ -1,11 +1,169 @@
 'use strict';
 
+// 1-1 Open the apps.js and start by declaring an array that contains 10 strings
+// These strings should be of book titles you have read (or made up) and be lowercase
+// without spaces so that you can use these later as HTML id attributes
+// (Example: Harry Potter's - The Chamber of Secrets -> harry_potter_chamber_secrets)
+// Add a console.log statement to output this array to console
+// (This is for debugging and making sure everything is in order
+// Delete it later when you're done :))
+
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+    'the-da-vinci-code',
+    'the-hunger-games',
+    'the-maze-runner',
+    'the-lord-of-the-rings',
+    'the-hobbit',
+    'a-game-of-thrones',
+    'harry-potter-and-the-goblet-of-fire',
+    'the-book-thief',
+    'angels-and-demons',
+    'harry-potter-and-the-order-of-the-phoenix',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // I have two functions, one to create a li items, and the second is to create ul element which is the parent of li
+  // I used for loop to loop through the array items, inside for loop I generate a list item
+  // I used document.createElement() method to create the HTML element,  which takes one parameter (tagName).
+  // I set the array elements which are string as a text content for li element using textContent property.
+  // I append all li elements to the ul by using append method to add a node to the given parent node parent.appendChild(child).
+  // For loop will loop as long as the condition is true.
+
```

Comment:
Hi Shyar - even though comments are helpful but in this case I think they are a bit too much. The code we write should more or less explain what the intention is of a function. In such cases, it is quite important that you name the functions correctly, have proper variable names and use more out of the box methods like Array.map, Array.filter or even the for loop which you are using. For example, your function names are quite understandable and explain what to expect out of them.

Reply:
I respect your point but from my perspective, even though my function names are quite understandable, but comments will help me later on when coming back to check my codes and understand what I exactly have done!